docs/grpc2.md

```diff
@@ -31,7 +31,8 @@ If these are enabled then the following options become available
 - `--grpc2-enable-grpc-web` (`CONCORDIUM_NODE_GRPC2_ENABLE_GRPC_WEB`) if set,
   enables the server support for [grpc-web](https://github.com/grpc/grpc-web)
   over HTTP 1.1. This allows the node's API to be used directly from a browser.
-- `--grpc2-health-max-finalized-delay` is a configuration for the
+- `--grpc2-health-max-finalized-delay`
```

Comment:
Maybe mention that the default setting is 5 minutes.

Reply:
Added.